google/cloud/storage/internal/async/connection_impl_test.cc

```diff
@@ -45,6 +45,7 @@ using ::testing::_;
 using ::testing::HasSubstr;
 using ::testing::Pair;
 using ::testing::UnorderedElementsAre;
+using ms=std::chrono::milliseconds;
```

Comment:
Consider moving this to the one function where it is used.

Reply:
Done